deepchem/models/tensorgraph/graph_layers.py

```diff
@@ -698,11 +699,13 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
       self.out_tensor = out_tensor
     return out_tensor
 
-  def DAGgraph_step(self, batch_inputs, W_list, b_list):
+  def DAGgraph_step(self, batch_inputs, W_list, b_list, **kwargs):
     outputs = batch_inputs
     for idw, W in enumerate(W_list):
       outputs = tf.nn.xw_plus_b(outputs, W, b_list[idw])
       outputs = self.activation(outputs)
+      if 'training' in kwargs and kwargs['training'] == 1.0 and not self.dropout is None:
```

Comment:
`training` will often be a tensor, not a number, so this comparison will fail.  Just multiply the dropout rate by it and don't worry about the value (like in the Dropout layer).

Reply:
Good point, just made the change!